### PR TITLE
Updating public API docs

### PR DESCRIPTION
**Pull request by julian-leung**
### What does this PR do?
Updating the public API docs to add new usage API endpoints and update existing API endpoints. 

### Motivation

New endpoint for logs-by-index created and available endpoints are currently not documented (Lambda, Synthetics API tests, etc...).

### Preview link
https://docs-staging.datadoghq.com/julian/usage-api-updates/api/?lang=bash#usage-metering

### Additional Notes
See the full list of changes here: 
https://docs.google.com/document/d/1ciu6grSlFrzQLLpnwjmXqeRuTV4eeEgwop90XHbzbRs/edit?usp=sharing

@decbis @carlmartensen do you mind reviewing?

fyi: @wissallahjouji

### CODE REVIEW

**Comment by carlmartensen** [comment]
The ordering is off in staging in that the main section doesn't match the curl example section, f/e click lambda

Screen Shot 2020-03-09 at 10 14 59 AM

and note the example is fargate

**Comment by julian-leung** [comment]
I had the older usage_synthetics_code.md file in there. Fixed it about 30 minutes ago. Changes are reflected now.

File: content/en/api/usage/usage_multi_org.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Get usage across your multi-org account
 type: apicontent
-order: 35.8
+order: 35.94
```
Comment (decbis) [inline content/en/api/usage/usage_multi_org.md]: nit/follow-up change: this should be updated as well to include the many other new & updated products made currently available [this API https://docs-staging.datadoghq.com/julian/usage-api-updates/api/?lang=bash#get-usage-across-your-multi-org-account]

Sample payload for org 2:
```
{
  "container_avg_sum": 17659,
  "last_updated": "2020-03-08T23",
  "alibaba_host_top99p_sum": 11,
  "aws_lambda_func_count": 85,
  "indexed_events_count_agg_sum": 103495049365,
  "infra_hours_agg_sum": 6168061,
  "netflow_indexed_events_count_agg_sum": 17148394531,
  "fargate_tasks_count_avg_sum": 2,
  "apm_host_top99p_sum": 20666,
  "usage": [
    {
      "custom_output_ts_avg": 110941613,
      "trace_search_indexed_events_count_sum": 36439530002,
      "custom_ts_avg": 329801673,
      "apm_host_top99p": 20666,
      "fargate_tasks_count_avg": 2,
      "infra_hours_sum": 6168061,
      "netflow_indexed_events_count_sum": 17148394531,
      "synthetics_browser_check_calls_count_sum": 770028,
      "aws_lambda_func_count": 85,
      "indexed_events_count_sum": 103495049365,
      "gcp_host_top99p": 58,
      "azure_host_top99p": 633,
      "synthetics_check_calls_count_sum": 13504537,
      "container_avg": 17659,
      "agent_host_top99p": 32773,
      "date": "2020-03",
      "npm_host_top99p": 20863,
      "infra_host_top99p": 33580,
      "ingested_events_bytes_sum": 360396895916192,
      "aws_host_top99p": 395,
      "alibaba_host_top99p": 11,
      "custom_input_ts_avg": 1306933
    }
  ],
  "custom_ts_sum": 329801673,
  "synthetics_check_calls_count_agg_sum": 13504537,
  "custom_output_ts_sum": 110941613,
  "end_date": "2020-03",
  "azure_host_top99p_sum": 633,
  "trace_search_indexed_events_count_agg_sum": 36439530002,
  "npm_host_top99p_sum": 20863,
  "ingested_events_bytes_agg_sum": 360396895916192,
  "synthetics_browser_check_calls_count_agg_sum": 770028,
  "agent_host_top99p_sum": 32773,
  "start_date": "2020-03",
  "aws_host_top99p_sum": 395,
  "gcp_host_top99p_sum": 58,
  "infra_host_top99p_sum": 33580,
  "custom_input_ts_sum": 1306933
}```

File: content/en/api/usage/usage_hosts.md [hunk 2]
```diff
@@ -34,3 +34,9 @@ Get Hourly Usage For Hosts and Containers.
 * **`aws_host_count`**:
     Contains the total number of hosts that reported via the AWS integration (and were NOT running the Datadog Agent).
     When AWS or GCP hosts are also running the Datadog Agent, they are counted as Agent hosts, NOT as AWS or GCP.
+* **`alibaba_host_count`**:
```
Comment (decbis) [inline content/en/api/usage/usage_hosts.md]: The `host_count` info needs to be updated as well [line 25 above https://github.com/DataDog/documentation/pull/6866/files#diff-2bd02b1dffb1190f6e72c9c1cca7466fR25] to mention that `alibaba_host_count` and `azure_host_count` are included as well.

File: content/en/api/usage/usage_hosts.md [hunk 3]
```diff
@@ -34,3 +34,9 @@ Get Hourly Usage For Hosts and Containers.
 * **`aws_host_count`**:
     Contains the total number of hosts that reported via the AWS integration (and were NOT running the Datadog Agent).
     When AWS or GCP hosts are also running the Datadog Agent, they are counted as Agent hosts, NOT as AWS or GCP.
```
Comment (decbis) [inline content/en/api/usage/usage_hosts.md]: nit: this seems superfluous at this point...

**Comment by carlmartensen** [comment]
All these usages should have double quotes
```
{
  usage: [
    {
```

**Pull request merged**